### ISSUE TITLE
Clarification on flag grad_norm_clip

### ISSUE THREAD

**Issue by kkhetarpal** [miyosuda/unreal#22]
When I try to run this codebase, the flag --grad_norm_clip =40.0 throws an error that 'Non-boolean argument to boolean flag' Could you clarify if this should be 40.0 or some bool value based on if we want clipped gradients or not?

Thanks

**Comment by miyosuda** [comment]
@kkhetarpal 
Sorry it is a bug and the value should be float, not boolean.

Please replace that line with.

> tf.app.flags.DEFINE_float("grad_norm_clip", 40.0, "gradient norm clipping")

(Maybe this bug became emergent by recent Tensorflow upgrade. I'll fix it. Thanks)

**Comment by kkhetarpal** [comment]
Yes, I fixed it my code with _float.
Thanks. Closing this issue.
